src/network/protocol/messages/version.zig

```diff
@@ -202,6 +204,24 @@ pub const VersionMessage = struct {
 
         return true;
     }
+
+    pub fn new(protocol_version: i32, me: protocol.NetworkAddress, you: protocol.NetworkAddress, nonce: u64, last_block: i32) Self {
+        return .{
+            .version = protocol_version,
+            .services = ServiceFlags.NODE_NETWORK,
+            .timestamp = std.time.timestamp(),
+            .recv_services = you.services,
+            .trans_services = me.services,
+            .recv_ip = you.ip,
+            .trans_ip = me.ip,
+            .recv_port = you.port,
+            .trans_port = me.port,
+            .nonce = nonce,
+            .user_agent = null,
+            .start_height = last_block,
+            .relay = null,
+        };
+    }
```

Comment:
I think this is better practice to put the default fields like `user_agent` or `relay` for ex as compile time known directly in the structure definition (default values know directly at compile time).

Reply:
done